x-pack/filebeat/input/httpjson/json_parser.go

```diff
@@ -0,0 +1,136 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package httpjson
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// parse returns array of string values from json string
```

Comment:
Here you could add some more details of what parse does, maybe add some examples. Because it's the entrypoint function to this whole JSON parsing logic, it's good to have it well documented.

Reply:
Sure. will add examples.